processing/src/main/java/io/druid/query/topn/TopNMapFn.java

```diff
@@ -33,6 +33,10 @@
 
 public class TopNMapFn
 {
+  public static final long ZERO_LONG = 0L;
```

Comment:
And more likely in the form of methods `nullToZero(Long)`, `nullToZero(Double)`, `nullToZero(Float)` directly, because those constants are not used in any other context.